### DISCUSSION TITLE
Pimcore 10 - system settings are not being saved

### DISCUSSION THREAD

**Discussion by damian1424** [pimcore/pimcore#10775] [Q&A / Support]
Hello everyone, 

I have a small problem with my system. 

In the system settings I want to add an error page. After saving the system settings, pimcore tells me that the settings have been saved successfully, then tells me that I should reload the page.  

After reloading the page, the "Error page" field in the system settings is empty. The same applies to other settings. 

Maybe I'm missing something important that needs to be activated, or set additionally?

I am not in debug mode.

![image](https://user-images.githubusercontent.com/32397559/140611481-bac2b0bc-964b-4b3f-ba99-4d8d711e331c.png)

**Comment by kopieMK** [comment]
Hey Damian,

Could you solve this problem? I have the same problem currently.

Greetings
Magnus

**Reply by dvesh3** [reply]
It stores the values in `/var/config/system.yml`, so it should be writable.

 Maybe check logs under `/var/log` to see if there's any error related to system settings save.

**Reply by kopieMK** [reply]
Divesh thanks for the quick feedback. 
I had already reset the permissions. But that had no positive effect. What solved it for me was that I ran the Composer update command again. After that it worked. But unfortunately I don't know what was fixed. 

I also have to clear the entire cache in the production system before I can see the values again in the backend (Settings > System Properties). This is a strange behavior but ok for now.
